src/helix-anon/index.html

```diff
@@ -61,25 +61,53 @@
       var webContext = VSS.getWebContext();
       var testRunId = tabContext["runId"];
       var testResultId = tabContext["resultId"];
-      
       var projectName = webContext.project.name;
       var organizationUrl = webContext.collection.uri;
 
       var getTestResultUrl = organizationUrl + projectName + "/_apis/test/Runs/" + testRunId + "/results/" + testResultId + "?api-version=6.0";
       window.fetch(getTestResultUrl)
         .then(response => response.json())
         .then(result => {
-          var data = JSON.parse(result.comment);
-          var jobId = data.HelixJobId;
-          var workItemName = data.HelixWorkItemName;
-          var link = "https://helix.dot.net/AzDevExtension/WorkItemResultData?jobName=" + encodeURIComponent(jobId) + "&workItemId=" + encodeURIComponent(workItemName);
+          var helixWorkItem = TryGetHelix(result.comment);
+          if (helixWorkItem.IsHelixWorkItem) {
+            try {
+              var jobId = helixWorkItem.HelixJobId;
+              var workItemName = helixWorkItem.HelixWorkItemName;
+              var link = "https://helix.dot.net/AzDevExtension/WorkItemResultData?jobName=" + encodeURIComponent(jobId) + "&workItemId=" + encodeURIComponent(workItemName);
 
-          window.location.href = link;
+              window.location.href = link;
+            } catch (e) {
+              document.getElementById("content").innerHTML = "<h3 style='color: red;'>Unable to Load Helix Data. Please contact <a href=\"mailto:dnceng@microsoft.com\">@dnceng</a>";
+            }
+          }
+          else {
+            document.getElementById("content").innerHTML = "<h3 style='color: red;'>This is not a helix test, for this reason there are no artifacts available in this tab</h3>";
```

Comment:
Super trivial, but maybe make this just normal black text. It's not an exceptional situation so "red" seems unnecessary.